uint16 isid
st_MVEVENT[] sign
int32 latitude
int32 longitude
uint16 speedlimit
uint16 count
